front/app/modules/commercial/project_description_builder/admin/components/ProjectDescriptionBuilderTopBar/ProjectDescriptionBuilderTopBar.test.tsx

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import { render, screen, fireEvent, act, within } from 'utils/testUtils/rtl';
-import ContentBuilderTopBar from './';
+import ProjectDescriptionBuilderTopBar from '.';
 import { Editor } from '@craftjs/core';
 import {
-  IContentBuilderLayoutData,
-  addContentBuilderLayout,
-} from '../../../services/contentBuilder';
+  IProjectDescriptionBuilderData,
+  addProjectDescriptionBuilderLayout,
+} from '../../../services/projectDescriptionBuilder';
 import clHistory from 'utils/cl-router/history';
 
-const mockEditorData: IContentBuilderLayoutData = {
+const mockEditorData: IProjectDescriptionBuilderData = {
   id: '2',
   type: 'content_builder_layout',
```

Comment:
Should we rename the type as well?

Reply:
I'm not sure... depends on how this is implemented in the backend. If this data structure is reusable for other content builders we can leave it like this, but if it's not it could also be renamed. WDYT @adessy ?

Reply:
Nope, it should stay as it is. All content builders will use the same type of layouts ✨